### PR TITLE
feat: Vue.js Integration TG-371

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

File: testapps/vue/src/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <T keyName="sampleApp.this_is_standard_text_somewhere_in_dom" />
+  </div>
+</template>
+
+<script lang="ts">
+import { TolgeeMixin, T } from '@tolgee/vue';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  mixins: [TolgeeMixin],
```
Comment (JanCizmar) [inline testapps/vue/src/components/Footer.vue:12]: This mixin

**Comment by stepan662** [comment]
Try to add language to mixin and use tolgeeContext only for tolgee instance

**Review by JanCizmar — COMMENTED** [review]
Please check if it runs with SSR frameworks

File: packages/vue/src/T.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    },
+    unsubscribe() {
+      this.$data.translationSubscription?.unsubscribe();
+      this.$data.langSubscription?.unsubscribe();
+    },
+    subscribe() {
+      // @ts-ignore
+      const tolgeeContext = this.tolgeeContext as TolgeeContext;
+
+      if (!this.$props.keyName) {
```
[the remark this answers is not in the document]
Comment (stepan662) [reply packages/vue/src/T.ts:57]: Vue can validate that natively, so I can just remove this

**Pull request merged**